hal/src/main/native/include/hal/HALBase.h

```diff
@@ -149,6 +149,13 @@ uint64_t HAL_ExpandFPGATime(uint32_t unexpanded_lower, int32_t* status);
  */
 HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode);
 
+/**
+ * Releases a waiting FPGA IRQ interrupt.
+ *
+ * @param interruptHandle the interrupt handle to release
+ */
+void HAL_ReleaseFPGAInterrupt(int32_t interruptNumber);
+
```

Comment:
Should there be a JNI wrapper for this function too?

Reply:
No. It's only exposed because of where the global struct is. It takes a raw interrupt number, which is only useful if you actually know the interrupt number.

Reply:
Should it not be a public function to C++ either then?

Reply:
I guess yeah I could move it to an internal header.